.github/CODEOWNERS

```
@@ -62,6 +62,7 @@
 /pubsub/**/*                           @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 /pubsublite/**/*                       @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 /managedkafka/**/*                     @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
+/dataplex/**/*                         @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers @GoogleCloudPlatform/googleapi-dataplex
```

same as above - we try to avoid folders without narrowed down owners.

---

But here `googleapi-dataplex` is specified as owner. Did you miss it or are you referring to something else?

---

```suggestion
/dataplex/**/*                         @GoogleCloudPlatform/googleapi-dataplex @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
```

I missed it, sorry :) Also, the smallest - most specific team should go first, them the more generic ones, as it's in this order automations look for approvers etc.